doc/source/api.rst

```diff
@@ -713,6 +714,7 @@ Indexing, iteration
    DataFrame.where
    DataFrame.mask
    DataFrame.query
+   DataFrame.sample
```

Comment:
I understand the duplication is intentional, but only adding to "... Selection ... " section may be better? I understand the function doesn't do indexing or iteration.